packages/cli/src/deploy/ensureResourceTags.ts

```diff
@@ -85,7 +83,7 @@ export async function ensureResourceTags<const value>({
       {
         optional: true,
         name: "MetadataModule",
-        installAsRoot: false,
+        installStrategy: "delegation",
```

Comment:
Do we want to install the metadata module via a delegation? (Do we want to give the installer ownership over the metadata tables or keep it like anyone can install it but it's immutable after it's been installed?)

Reply:
chatted in discord:

this doesn't change the current behavior, which passed namespace ownership back to the deployer: https://github.com/latticexyz/mud/pull/3586/files#diff-4b5b7ac0dc35d7914ce67b4dfcee8d54a4814b759a87d0ddce84efe17d3ade08L55

can separately decide what we wanna do going forward